.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+package-lock.json
```

Comment:
I am curious: this project uses yarn right now, is this just to make sure it's not added as an mistake?

Reply:
Yes, it's my mistake, fixed. 
Thanks